exception_notifier.gemspec

```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |s|
   s.name = 'exception_notifier'
-  s.version = '0.1.5'
+  s.version = '0.1.6'
```

Comment:
Should not break anything in charity api. However, we WILL need to update the config per their migration docs:

https://docs.sentry.io/platforms/ruby/configuration/options/

TLDR
```ruby
# from
Raven.configure do |config|
...
end

# to
Sentry.init do |config|
...
end
```